Formula/indel-seq-gen.rb

```diff
@@ -14,7 +14,6 @@ class IndelSeqGen < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
-  # Fix error: random.cpp:148:5: error: reference to 'next' is ambiguous
```

Comment:
I'd like to have this comment back in, to explain the reason for the `depends_on :linux`

Reply:
whoops didnt mean to kill that